### ISSUE TITLE
On Mac, Framegrab will find your iPhone as a valid camera

### ISSUE THREAD

**Issue by sunildkumar** [groundlight/framegrab#24]
Discussed this offline - It would be awesome if framegrab autodiscover had some metadata to help differentiate between cameras in cases like this, as right now they are indistinguishable: 

```bash
# Auto-discovered 2 camera(s).
image_sources:
- input_type: generic_usb
  name: GENERIC_USB Camera - 1
  options: {}
- input_type: generic_usb
  name: GENERIC_USB Camera - 2
  options: {}
```

**Comment by robotrapta** [comment]
I agree this is annoying, but I'm not sure what we can do here.  It seems like Apple is pushing this on everybody.  e.g. Slack huddles defaults to the iPhone camera if it's configured through iCloud.
